pkg/storaged/dialogx.jsx

```diff
@@ -701,12 +701,13 @@ class SizeSliderElement extends React.Component {
                     value = limit;
             }
 
+            sanitize(min);
             sanitize(max);
 
             return Math.round(value);
         };
 
-        const change_slider = (f) => onChange(round_size(f * max));
+        const change_slider = (f) => onChange(round_size(Math.max(min, f * max)));
```

Comment:
That looks curious to me -- if we assume that max > min, then this should never effectively change anything. Wouldn't it make more sense to pass the minimum to the rounding function? Or does that really guard against the min > max case?

Reply:
This simply restricts the value of the slider to be >= min.  If f*max < min, then Math.max(min, f*max) = min.  I don't think this clamping is done elsewhere, which would make this code here redundant.  Did I misunderstand your comment?

Reply:
I just wasn't sure if you really intend to do what it currently does. It seemed more logical that the slider would be clamped between min and max, and that min < max would be asserted much earlier on. I mean, this is the change handler, so if that ever actually returns a value that is < min or > max, then something went seriously wrong, no?

Reply:
A StatelessSlider always goes between 0 and 1, and a SizeSliderElement takes care of min, max, rounding, textual input, units, and infinity.  Did you expect that the StatelessSlider would take care of min, and the SizeSliderElement of the rest?

That would certainly be possible, but since the SizeSliderElement already munges the value so much, letting it clamp the value at min also makes sense to me.